core/src/main/java/com/github/mizool/core/configuration/EnvironmentVariableConfigurationImpl.java

```diff
@@ -0,0 +1,26 @@
+package com.github.mizool.core.configuration;
+
+import javax.enterprise.inject.Vetoed;
+
+import com.google.common.base.Strings;
+
+@Vetoed
+public class EnvironmentVariableConfigurationImpl implements Configuration
+{
+    @Override
+    public String get(String key)
+    {
+        return System.getenv(key);
+    }
+
+    @Override
+    public String get(String key, String def)
+    {
+        String result = System.getenv(key);
+        if (Strings.isNullOrEmpty(result))
```

Comment:
This is not syntactically correct.
`System.getProperty(String key, String def)` will _only_ return the default value when the system property is not defined. If it's defined but empty, an empty string is returned.
Since `System.getenv(String name)` will also return an empty string if the environment variable is defined but empty and only return `null` when the environment variable is not defined, we have to check for `result == null` to imitate the behavior of `System.getProperty(String key, String def)`.